roles/kubernetes-apps/scheduler_plugins/templates/kube-scheduler-patch.yq.j2

```diff
@@ -0,0 +1,8 @@
+{# Step 6 in https://github.com/kubernetes-sigs/scheduler-plugins/blob/v0.27.8/doc/install.md#as-a-single-scheduler-replacing-the-vanilla-default-scheduler #}
```

Comment:
Are there more elegant ways to patch besides using yq? 🤔

Reply:
I don't think there is such a better way. `kubectl` can only change the non-static pod, static pod must be changed by modifying the manifest file within the `/etc/kubernetes/manifests`. And those are `YAML` files, changing them by `yq` is the appropriate choice.

Reply:
That's correct, the main effective action is `sleep infinity`, but users may also want to know the specific cluster LCM Ansible operation commands by viewing the `entrypoint.sh` script file.

Reply:
Can't we instead work in `kubernetes/control-plane` to generate the manifests correctly once ? I think this would change the scheduler twice per upgrade, which is pretty bad.

Reply:
I've considered your point, but I'm a bit hesitant to go that way, as this module would be strongly coupled with the `kubernetes/control-plane` code. Do you think that is still the better option?

Reply:
Well, scheduler-plugins are tightly coupled with the scheduler anyway.
Another option (besides putting it directly in our templates) could be to use [kubeadm patches](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/control-plane-flags/#patches) , (would be better than yq, as we would not modify the manifest multiples times on upgrade)